docs/general/adapters.md

```diff
@@ -117,46 +117,46 @@ The default adapter can be configured, as above, to use any class given to it.
 
 An adapter may also be specified, e.g. when rendering, as a class or as a symbol.
 If a symbol, then the adapter must be, e.g. `:great_example`,
-`ActiveModel::Serializer::Adapter::GreatExample`, or registered.
+`ActiveModelSerializers::Adapter::GreatExample`, or registered.
 
 There are two ways to register an adapter:
 
-1) The simplest, is to subclass `ActiveModel::Serializer::Adapter::Base`, e.g. the below will
+1) The simplest, is to subclass `ActiveModelSerializers::Adapter::Base`, e.g. the below will
 register the `Example::UsefulAdapter` as `"example/useful_adapter"`.
```

Comment:
It's  `"example/useful_adapter"`, though it can be looked up as `:"example/useful_adapter" if you want

```ruby
# code that registers the adapter `name` to the `klass`
 name = name.to_s.gsub(/\AActiveModel::Serializer::Adapter::/, ''.freeze)
 adapter_map.update(name.underscore => klass)
```